airflow/models/dag.py

```diff
@@ -1386,6 +1386,7 @@ def get_template_env(self, *, force_sandboxed: bool = False) -> jinja2.Environme
             "extensions": ["jinja2.ext.do"],
             "cache_size": 0,
         }
+        self.log.warning("SEARCHPATH: %s", searchpath)
```

Comment:
Is this a testing leftover or what is this change for?

Reply:
aaaaaaaaaa.... Stupid me. 

Glad that the PR is now "somewhat" reviewable". And that you did it ! 

Let me re-review it myself before next push.